test/suites/dev/moonbase/test-staking/test-staking-locks10.ts

```diff
@@ -8,11 +8,14 @@ import {
   baltathar,
   generateKeyringPair,
 } from "@moonwall/util";
-import { fromBytes } from "viem";
+import {
+  getDelegatorStakingFreeze,
+  getNumberOfDelegatorFreezes,
+} from "../../../../helpers/staking-freezes";
 
 describeSuite({
-  id: "D023475",
-  title: "Staking - Locks - multiple delegations single lock",
+  id: "D013473",
```

Comment:
This change was also caused by a conflict, we should revert it to keep the test id order.